src/libs/ReportUtils.ts

```diff
@@ -8433,6 +8433,14 @@ function hasInvoiceReports() {
     return allReports.some((report) => isInvoiceReport(report));
 }
 
+function areAllGroupPoliciesExpenseChatDisabled(policies = allPolicies) {
```

Comment:
Thanks for adding! I think this would fit better in PolicyUtils though.

Reply:
PolicyUtils is also already connected to all policies, so I don't think you'd need to pass the policies to it.

Reply:
ah cool, good to know